Start-Your-Engine/src/GameObject.cpp

```diff
@@ -76,7 +179,13 @@ glm::vec2 GameObject::getPosition() {
 	std::cout << "\n" << std::endl;
 	*/
 	
-	return glm::vec2(this->body->GetPosition().x - ((this->size.x) / 2), this->body->GetPosition().y - ((this->size.y) / 2));
+	return glm::vec2(this->body->GetPosition().x - ((this->size.x) / 2)
+		, this->body->GetPosition().y - ((this->size.y) / 2));
+}
+
+glm::vec2 GameObject::getLocation()
```

Comment:
should add a comment to say how this is different from getPosition(). Seems like this returns the position of the physics body instead of the render position.